### PR TITLE
Add margin between buttons by 8x, total 24px

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
-  Adjust spacing between the action items in the toolbar.

This PR only adds 8px more padding between action-items. 

image

### CODE REVIEW

**Comment by emclaug2** [comment]
Is this the ideal spacing between the title and menu icon?  Seems tight to me.

image

**Comment by huayunh** [comment]
https://px-blue.github.io/patterns/appbar#design-specifications

**Comment by emclaug2** [comment]
@joebochill  @huayunh 

This PR is updated now to follow the pxb appbar spec;  React PR is being updated right now.

File: src/styles.scss [hunk 1]
```diff
@@ -21,6 +21,8 @@ body {
     position: sticky;
     top: 0;
     z-index: 1000;
+    box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2), 0px 4px 5px 0px rgba(0, 0, 0, 0.14),
+        0px 1px 10px 0px rgba(0, 0, 0, 0.12);
```
Comment (huayunh) [inline src/styles.scss:25]: https://material.angular.io/guide/elevation

Comment (emclaug2) [reply src/styles.scss:25]: I'm going to add class `mat-elevation-z8` to each individual toolbar.

Comment (emclaug2) [reply src/styles.scss:25]: `z8` is way too much elevation.   I think `z2` works.

Comment (emclaug2) [reply src/styles.scss]: Collapsible appbar already has z2 elevation on it, I think this is the correct size.

**Review by joebochill — APPROVED** [review]
Collapsible Appbar (large version, mobile viewport) menu icon is off by 4px (should be -8)

**Comment by emclaug2** [comment]
> Collapsible Appbar (large version, mobile viewport) menu icon is off by 4px (should be -8)

Good catch, this is updated to -8px now.

**Pull request merged**